### DISCUSSION TITLE
Add data rate sensor device class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#834] [Entity Models] [answered]
I propose that we add a `datarate` sensor device_class to home assistant core and frontend (+ documentation) with supported units:
- `***BITS_PER_SECOND`
- `***BYTES_PER_SECOND`
- `***BIBYTES_PER_SECOND`

Just looking at `DATA_RATE_MEGABITS_PER_SECOND` unit, all these integrations would benefit: `bbox`, `fastdotcom`, `iperf3`, `netgear`, `nzbget`, `pyload`, `sabnzbd`, `speedtestdotnet`, `systemmonitor`, `transmission`, `unifi`

Implementation PR: https://github.com/home-assistant/core/pull/83377

**Comment by frenck** [comment, marked as answer]
Makes sense 👍

**Comment by MartinHjelmare** [comment]
I agree. For completeness please count the number of integrations that can benefit.

**Reply by epenet** [reply]
Edited the description to add some of them (too many to list all)

**Reply by MartinHjelmare** [reply]
How many is it in total?

**Reply by epenet** [reply]
23 components use `DATA_RATE_***` units
